common/templates/common/docs/catalog.html

```diff
@@ -10,13 +10,92 @@ <h2 class="red-text text-darken-4">Catalog</h2>
+  <li class="collection-item"><span class="code">public</span> - whether the subject is a custom subject. For catalog subjects, this is always false.</li>
+</ul>
+
+<p>These dictionaries may optionally contain the keys:</p>
+
+<ul class="collection">
+  <li class="collection-item"><span class="code">level</span> - undergraduate or graduate. Possible values: "U", "G".</li>
+
+  <li class="collection-item"><span class="code">is_historical</span> - whether the subject is historical, i.e. not in the current catalog.</li>
+  <li class="collection-item"><span class="code">source_semester</span> - for historical subjects, the semester info is sourced from, e.g. "fall-2018".</li>
+
+  <li class="collection-item"><span class="code">joint_subjects</span> - list of equivalent subject IDs, e.g. ["18.062"].</li>
+  <li class="collection-item"><span class="code">equivalent_subjects</span> - list of subject IDs that credit cannot also be received for, e.g. for 8.02, ["8.021", "8.022", "ES.802", "ES.8022"].</li>
+  <li class="collection-item"><span class="code">meets_with_subjects</span> - list of subject IDS that the subject meets with.</li>
+
+  <li class="collection-item"><span class="code">quarter_information</span> - string representing when subject ends or begins, e.g. "0,apr 14" meaning subject ends on Apr 14, or "1,4/4" means subject begins on 4/4.</li>
+  <li class="collection-item"><span class="code">not_offered_year</span> - school year subject is not offered in, e.g. "2022-2023".</li>
+
+  <li class="collection-item"><span class="code">instructors</span> - list of instructors. Each entry represents a semester, and multiple instructors can appear in a single semester, e.g. ["Spring: A. Hacker", "Fall: A. Hacker, B. Bitdiddle"].</li>
+  <li class="collection-item"><span class="code">communication_requirement</span> - possible values: "CI-H", "CI-HW".</li>
+  <li class="collection-item"><span class="code">hass_attribute</span> - comma-separated string, with possible values: "HASS-A", "HASS-E", "HASS-H", "HASS-S".</li>
+  <li class="collection-item"><span class="code">gir_attribute</span> - possible values: "BIOL", "CAL1", "CAL2", "CHEM", "LAB", "LAB2", "PHY1", "PHY2", "REST".</li>
+
+  <li class="collection-item"><span class="code">children</span> - for subjects that meet with several classes per semester, e.g. ["5.601", "5.602"].</li>
+  <li class="collection-item"><span class="code">parent</span> - parent subject IDs for subjects that are children, e.g. for 5.601 or 5.602, "5.60".</li>
+
+  <li class="collection-item"><span class="code">virtual_status</span> - possible values: "In-Person", "Virtual".</li>
+  <li class="collection-item"><span class="code">old_id</span> - old subject ID, e.g. "6.009".</li>
```

Comment:
I would change 6.009 to 6.042 to align with the example of 6.1200 (because it's possible to do that here).

Reply:
oh i hate these old numbers